docs/configuration.rst

```diff
@@ -449,17 +453,29 @@ remain available as described at the start of the :ref:`Configuration` section.
     other operation in your `Magpie` instance. The application will quit with an error if this value cannot be found.
     It is recommended that the developer configures every new instance with server-specific and strong credentials.
 
+    Prior versions also allowed modification of this value from the API and UI, which increased chances of having
+    out-of-sync definitions between the database and :term:`Configuration` files. This is not permitted anymore.
```

Comment:
Agree with this problem "out-of-sync definitions between the database and :term:`Configuration` files"